src/hooks/spaces/useSpacesByOwner.tsx

```diff
@@ -35,7 +33,11 @@ export const useSpacesByOwner: UseSpacesByOwner = ({ worldId, userId }) => {
     [worldId, userId]
   );
 
-  const result = useLiveCollection<SpaceWithId>({ path, constraints });
+  const options = useMemo(() => ({ path: PATH.spaces, constraints }), [
```

Comment:
👍 Nice find. I will have to update some hook APIs to make it harder to fall into this trap in the future.